### PR TITLE
Replace failing tests

### PR DESCRIPTION
**Pull request by lily-pytel**
Can't believe this but it seems like even though `not.be.visible` works locally, on Travis the element actually doesn't exist??? So it looks like `not.exist` does work

### CODE REVIEW

**Comment by lily-pytel** [comment]
Son of a 🔫 🔫 🔫 🔫 🔫 🔫 🔫

**Comment by lily-pytel** [comment]
I am baffled as to why `not.be.visible` _used to work_ and not it doesn't. And why it works in development. Did Travis update? Do we run in dev mode locally? Is this just a cruel universe?

**Comment by alexmiddeleer** [comment]
Wow can't believe the tests passed!

File: .travis.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ before_install:
 install:
   # Install node modules (for jest) and start up the docker containers
   - cd next
-  - npm ci
+  - npm install
```
Comment (alexmiddeleer) [inline .travis.yml:36]: why did we change this from ci? I think we should keep using ci....

File: e2e/cypress/integration/deck-builder-page.js [hunk 2]
```diff
@@ -27,9 +27,9 @@ describe('Deck builder page', () => {
   it('should have a happy path', function() {
     cy.get('[data-cy="header"]').should('be.visible');
     cy.get('[data-cy="goToImportMode"]').should('be.visible');
-    cy.get('[data-cy="importDeckButton"]').should('not.be.visible');
-    cy.get('[data-cy="deckBuilderActions"]').should('not.be.visible');
-    cy.get('[data-cy="deckManaCurve"]').should('not.be.visible');
+    cy.get('[data-cy="importDeckButton"]').should('not.exist');
```
Comment (alexmiddeleer) [inline e2e/cypress/integration/deck-builder-page.js:30]: Hmm, strange. Maybe cypress got upgraded and we didn't notice. Seems like a dumb breaking change though.

File: express/Dockerfile [hunk 3]
```diff
@@ -4,7 +4,7 @@ WORKDIR /app
 
 COPY . /app
 
-RUN npm ci --production
+RUN npm i --production
```
Comment (alexmiddeleer) [inline express/Dockerfile:7]: ditto I think we should keep using ci unless tests don't pass. If they don't pass I'm guessing we should try to figure out which version we need and keep using npm ci so it's less brittle.

**Review by alexmiddeleer — APPROVED** [review]
Hopefully the lib upgrades don't break production again but I'll roll back if they do. It's easy. Thanks so much for figuring out the tests, I was super stumped and ready to give up.

File: e2e/cypress/integration/deck-builder-page.js (hunk 2 above)
Comment (lily-pytel) [reply e2e/cypress/integration/deck-builder-page.js:30]: Very dumb!!! Can't believe this made the tests green

File: .travis.yml (hunk 1 above)
Comment (lily-pytel) [reply .travis.yml:36]: It looked like when ci was used, the tests failed in a different way but let me try bringing it back

**Comment by lily-pytel** [comment]
Wow bringing back `npm ci` was _not_ well received by travis

**Comment by lily-pytel** [comment]
What do you think about merging this? It's at least green and we can treat it as a starting point to bringing back `npm ci`.

**Comment by alexmiddeleer** [comment]
@lily-pytel sounds good. Thanks again!

**Pull request merged**